test/test-service.js

```diff
@@ -1,4 +1,4 @@
-var should = require('should');
+require('should');
```

Comment:
how can this module be used if not named ?
maybe it should not be imported in the first place ?

Reply:
[It hooks up `.should.` stuff globally on all objects.](https://github.com/shouldjs/should.js/blob/690405e879d2a46c3bac38fbfabe1dce3c7de44d/should.js#L4349-L4361)